Quick Start Documentation/Instrument Addon Quick Start Guide.md

```diff
@@ -0,0 +1,290 @@
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\Host Automation API<br/><br/>
+An example of using this API is as follows:
+
+![Host API Example](./Images/image036.png)
+
+First, connect to the add-on by its name and expected target IP address. Then send raw messages. When finished, close the connection.
+
+## System Definition API
+
+There is also a system definition API that allows the user to programatically create an instance of the add-on from a previously exported configuration.<br/>
+This is installed at:<br/>
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\System Definition API<br/><br/>
+An example of using this API is as follows:​​​
+
+![System Definition API Example](./Images/image038.jpg)
+
+## Scripting API
+
+Users can add flexibility and reusability to their systems by creating and configuring system definitions through the Scripting API.
+
+## With this API, users can:<br/>
+- create a new Instrument from a previously exported configuration:
+
+    Node used: **Create Instrument Addon From Configuration File**
+
+![Create Instrument Addon From Configuration File](./Images/image040.png)
+
+- create a new Instrument from scratch<br/>
+    Node used: **Create Instrument Addon**
+
+![Create Instrument Addon](./Images/image041.png)
+
+- add Messages and configure their Command/Response 
+
+    Node used: **Create Message Group**<br/>
+![Create Message Group](./Images/image042.png)
+
+    Node used: **Add Command**<br/>
+![Add Command](./Images/image043.png)
+
+    Node used: **Add Response**<br/>
+![Add Response](./Images/image044.png)
+
+    Node used: **Create Message**<br/>
+
+![Create Message](./Images/image045.png)
+
+- Create Channels
+
+    Node used: **Create Global MetaData Channel**<br/>
+![Create Global MetaData Channel](./Images/image046.png)
+
+    Node used: **Create MetaData Channel**<br/>   
+![Create MetaData Channel](./Images/image047.png)
+
+    Node used: **Create Data Channel**<br/>
+![Create Data Channel](./Images/image048.png)
+
+- Fetch the Protocol object for a specific Instrument Addon
+
+    Node used: **Get Protocol**
+   
+- Change the Protocol object (all or specific properties)
+
+    Node used: **Set Protocol**   
+
+![Get and Set Protocol](./Images/image049.png)
+
+- Get and Set the PCL Decimation
+
+    Node used: **Get PCL Decimation**
+    Node used: **Set PCL Decimation**   
+
+![Get and Set PCL Decimation](./Images/image050.png)
+
+- Make use of the Protocols subpalette - containing the Get Protocol Type node and factory nodes for each supported protocol
+   
+A Protocol object is needed as input when creating a new Instrument Addon<br/>
+List of supported protocols:<br/>
+   - RS-xxx
+   - RS-xxx to ConvBox
+   - VISA
+   - TCP
+   - UDP
+   - AK RS-xxx
+   - AK TCP
+   
+    Factory Nodes used:
+
+![Factories](./Images/image052.png)   
```

Comment:
For whatever reason the above line is rendered different than the other lines. This breaks the consistency of the file.

![image](https://user-images.githubusercontent.com/62251693/94682284-5fc3a100-032d-11eb-9c2d-dd38359634c6.png)